lib/output.js

```diff
@@ -887,6 +904,21 @@ function OutputStream(options) {
         self._do_print(output, "throw");
     });
 
+    /* -----[ yield ]----- */
+
+    DEFPRINT(AST_Yield, function(self, output){
+        var star = self.is_star ? "*" : "";
+        output.print("yield" + star);
+        if (self.expression) {
+            output.space();
+            self.expression.print(output);
+        } else if (self.is_star === true) {
```

Comment:
I believe we should assume that any AST we consume is valid. We could go on with small exceptions to the rule. ;-)

(I know this is nitpicking, but it's important to me to keep things simple)